digital-twin-model/src/sdv_v1.rs

```diff
@@ -2,30 +2,52 @@
 // Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
 
-// Note: In the future this code could be auto-generated from a DTDL spec.
+// Note: In the future this code should be generated from a DTDL spec.
 
-pub mod vehicle {
-    pub mod cabin {
-        pub mod hvac {
-            pub mod ambient_air_temperature {
-                pub const ID: &str = "dtmi:sdv:Vehicle:Cabin:HVAC:AmbientAirTemperature;1";
-            }
-            pub mod is_air_conditioning_active {
-                pub const ID: &str = "dtmi:sdv:Vehicle:Cabin:HVAC:IsAirConditioningActive;1";
-            }
-        }
-        pub mod infotainment {
-            pub mod hmi {
-                pub mod show_notification {
-                    pub const ID: &str =
-                        "dtmi:sdv:Vehicle:Cabin:Infotainment:HMI:ShowNotification;1";
-                }
-            }
-        }
+pub mod airbag_seat_massager {
```

Comment:
Is there a way to have these linked to the DTDL so that they don't need to be maintained separately? Perhaps we should move away from this model with constants and instead use the DTDL file as the source of truth with a library to provide read and access methods which mimic these modules.

Alternatively, maybe we use modules like this as the source of truth and auto-generate the DTDL from them

Reply:
The ideal solution is to generate this code from the DTDL.  I have a comment in this file saying that this needs to be done at some point.  I will do it when it becomes my highest priority.  I can use the C# DTDL parser to build the tool to do this.